lib/rubocop/cop/performance/count.rb

```diff
@@ -62,6 +76,15 @@ def autocorrect(node)
 
         private
 
+        def should_run?
+          if cop_config['SafeMode'.freeze]
```

Comment:
Is the `if` necessary here - seems to me the condition already returns `true` or `false`.